src/test/java/org/jenkinsci/plugins/mesos/integration/MesosApiTest.java

```diff
@@ -41,11 +43,19 @@
   static ActorSystem system = ActorSystem.create("mesos-scheduler-test");
   static ActorMaterializer materializer = ActorMaterializer.create(system);
 
+  static MesosAgentConfig config =
+      new MesosAgentConfig(
+          "linux",
+          "mesos",
+          Option.apply("filesystem/linux,docker/runtime"),
```

Comment:
Is this working on macOS?

Reply:
parameterized to only run docker test in linux